src/pages/settings/tax-settings/TaxSettings.tsx

```diff
@@ -126,10 +130,35 @@ export function TaxSettings() {
             )}
           </div>
         </Element>
-      </Card>
 
-      <Selector />
+        {usePaidOrSelfHost() && useCalculateTaxesRegion(companyChanges?.settings?.country_id) ? (
```

Comment:
Same as with the previous example.

{ usePaidOrSelfHosted && useCalculateTaxesRegion && <> } should do the job.